### PR TITLE
Separate Larastan config

### PR DESCRIPTION
**Pull request by szepeviktor**
@mfn What do you think?

### CODE REVIEW

File: phpstan.neon.dist [hunk 1]
```diff
@@ -1,23 +1,13 @@
+includes:
+    - larastan.neon
```
Comment (mfn) [inline phpstan.neon.dist]: I'm wondering, can't we just reference the config _shipped_ with larastan that way? 🤷‍♀️

Comment (szepeviktor) [reply phpstan.neon.dist]: Feel free to experiment with it. Actually I promote phpstan, I'm not a graphql user.

Comment (szepeviktor) [reply phpstan.neon.dist]: `vendor/nunomaduro/larastan/extension.neon`

Comment (szepeviktor) [reply phpstan.neon.dist]: I do not know how to _un-ignore_ errors.

Comment (mfn) [reply phpstan.neon.dist]: > I do not know how to _un-ignore_ errors.

Ahh. I guess that what the reason in the first place why I assimilated it.

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**